frontend/src/lib/components/details/seasons.svelte

```diff
@@ -1,124 +1,64 @@
-              {#if season.poster_path}
-                <figure>
-                  <img
-                    src="{IMG_W500}{season.poster_path}"
-                    class="object-fit h-96"
-                    alt={season.name}
-                  />
-                </figure>
-              {:else}
-                <figure>
-                  <img
-                    src="../../no_image_available.jpg"
-                    class="object-fit h-72"
-                    alt="No poster available"
-                  />
-                </figure>
-              {/if}
-              <div class="card-body">
-                <div class="card-title">{season.name}</div>
-                <div class="text-xl text-netural-content">
-                  {season.air_date ? season.air_date.split("-")[0] : "No air date"}
-                  | {season.episode_count} episodes
-                </div>
-                <div class="text-lg text-netural-content">
-                  {season.air_date ? `Premiered on ${season.air_date}` : "Hasn't aired"}
-                </div>
-                &nbsp
-                <div class="text-netural-content">
-                  {season.overview ? season.overview : "No season overview available."}
-                </div>
-              </div>
-            </div>
-          {/if}
-        {/if}
-      </MediaQuery>
-      <MediaQuery query="(min-width: 1025px)" let:matches>
-        {#if matches}
-          {#if index === currentTab}
-            <div class="card card-side bg-neutral bordered lg:w-5/6 xl:w-3/5">
-              {#if season.poster_path}
-                <figure>
-                  <img
-                    src="{IMG_W342}{season.poster_path}"
-                    class="object-fit h-96"
-                    alt={season.name}
-                  />
-                </figure>
-              {:else}
-                <figure>
-                  <img
-                    src="../../no_image_available.jpg"
-                    class="object-fit h-72"
-                    alt="No poster available"
-                  />
-                </figure>
-              {/if}
-              <div class="mx-4 my-2">
-                <div class="card-title">{season.name}</div>
-                <div class="text-xl text-netural-content">
-                  {season.air_date ? season.air_date.split("-")[0] : "No air date"}
-                  | {season.episode_count} episodes
-                </div>
-                <div class="text-lg text-netural-content">
-                  {season.air_date ? `Premiered on ${season.air_date}` : "Hasn't aired"}
-                </div>
-                &nbsp
-                <ReadMore
-                  currentDescriptionLength={currentOverviewLength}
-                  mediaDescription={season.overview
-                    ? season.overview
-                    : "No season overview available."}
-                  initialDescriptionLength={INITIAL_OVERVIEW_LENGTH}
+  <div class="swiper-container">
```

Comment:
Can we add `cursor pointer`?

Reply:
<img src="https://media0.giphy.com/media/kaBmqpJtdvFqo/giphy.gif"/>